### PR TITLE
Update creeperita104.json and delete other nested subdomains

### PR DESCRIPTION
**Pull request by creeperita09**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

I'll upload a screenshot of my site in a comment since rn I'm on mobile so i cannot multitask, my reasons for using ns are that i need ddos protection and i would like my home ip address to not be public (my site is running out of nginx on a pi4), it would also make it easier for me using cloudflare's dashboard.

### CODE REVIEW

File: domains/creeperita104.json [hunk 1]
```diff
@@ -3,7 +3,5 @@
     "username": "creeperita09",
     "email": "creeperita.09@gmail.com"
   },
-  "record": {
-    "CNAME": "icahomesvr2022.freeddns.org"
-  }
+  "NS":["kyrie.ns.cloudflare.com","vida.ns.cloudflare.com"]
```
Comment (Satindar31) [inline domains/creeperita104.json:6]: Invalid format. Do 
```suggestion
  "record": {
    "NS":["kyrie.ns.cloudflare.com","vida.ns.cloudflare.com"]
   }
```

**Review by Stef-DP — DISMISSED** [review]
what @Satindar31's said ^

**Comment by creeperita09** [comment]
Sorry my bad, apparently i was high on something

**Comment by creeperita09** [comment]
Ok, now it should be valid, sorry for wasting your time

**Review by wdhdev — APPROVED** [review]

**Pull request merged**